### ISSUE TITLE
✅ 🐞 Bug Report: Elementor pro spoils wp-cli ouput with message about bad types in some cases - [ED-10136]

### ISSUE THREAD

**Issue by d1sparity-plesk** [elementor/elementor#21582]
### Prerequisites

- [X] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [X] The issue still exists against the latest stable version of Elementor.

### Description

From time to time Elementor spoils output with records about bad types on PHP 8.1.

The same error message is here: https://github.com/elementor/elementor/issues/21460 but here is another case.

**actual:**
```
string(56) "https://domain.tld/wp-login.php"
PHP: 2023-02-28 14:58:38 [notice X 0][/path-to-wordpress/wp-content/plugins/elementor-pro/modules/loop-builder/module.php::200] strpos(): Passing null to parameter #1 ($haystack) of type string is deprecated [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
```

**epected**
```
string(56) "https://domain.tld/wp-login.php"
```

```
PHP: 8.1.16
WP-CLI: 2.6.0
WordPress: 6.1.1
Elementor: 3.11.2
Elementor Pro: 3.11.1
```

### Steps to reproduce

1. Obtain WordPress + wp-cli + elementor plugins on PHP 8.1 (according to the versions)
2. Run several times this command
`wp --no-color --skip-plugins=false eval 'var_dump(wp_login_url());'`
the error appears from the second+ time.

### Isolating the problem

- [ ] This bug happens with only Elementor plugin active (and Elementor Pro).
- [ ] This bug happens with a Blank WordPress theme active ([Hello theme](https://wordpress.org/themes/hello-elementor/)).
- [ ] I can reproduce this bug consistently following the steps above.

### System Info

-

**Comment by austinginder** [comment]
Same issues on a few of my customer websites. I confirmed that downgrading from PHP 8.1 to PHP 8.0 is a workaround.

<img width="1399" alt="Screenshot 2023-03-08 at 9 37 58 AM" src="https://user-images.githubusercontent.com/110346/223742030-648689a3-627b-43c4-82aa-fe506915ca04.png">

**Comment by VerdiH** [comment]
@nicholaszein Pleask reached out directly about this, please make sure it has the proper attention 🙏

**Comment by nicholaszein** [comment]
Hello @d1sparity-plesk! 

Thank you for submitting your issue! 🙏 

We are investigating this, and will update you when we have more information! 😉 

Best regards

**Comment by Netzlichter** [comment]
Can you please provide a fix? I have that problem over a month now... Thank you :)!

**Comment by nicholaszein** [comment]
Hello everyone!

We have great news! 🙌 

### 📢 We're happy to announce that the issue you raised was resolved in Elementor Pro **v3.12.3**! 🥳 

✅ Feel free to check it out and update your plugin to the new version!

Cheers 🥂
